docs/docs/porting-to-vial.md

```diff
@@ -14,37 +14,78 @@ redirect_from:
-2. Run `make git-submodule` in your new `vial-qmk` directory to clone the git submodules.
-3. Continue to run `make path/to/your/keyboard:keymap` for Vial builds from this directory. Make sure the `default` keymap for your keyboard compiles successfully. For example, if your keyboard is located in the `keyboards/xyz/xyz60` folder, to compile it using the `default` keymap, type `make xyz/xyz60:default`.
+3. Verify that your installation is complete by running `qmk doctor`.
+   - If you have added a new keyboard folder as part of your port, you will see `Git has unstashed/uncommitted changes.` Other than that, the only warning should be `The official repository does not seem to be configured as git remote "upstream".` (This is fine because Vial-QMK is not the official QMK repository.)
+4. Test your Vial-QMK installation by compiling the `default` keymap for your keyboard.
+   - Run `make path/to/your/keyboard:default`. Note that the path does *not* include the folder `vial-qmk/keyboards/` nor does it have a leading slash. It should start with the name of the keyboard or designer, e.g. `make vial_example/vial_atmega32u4:default`
+     If the `default` keyboard doesn't build, nothing based on it will either! Code copy-and-pasted in from other repositories may need to be modified [based on the version of QMK it was created for](https://docs.qmk.fm/#/breaking_changes?id=what-has-been-included-in-past-breaking-changes).
+
+> Note
+> {: .label .label-green }
+> Running `make` is not quite the same as running `qmk compile`. Running `qmk compile` in the wrong directory will default to compiling QMK firmware, not Vial firmware. To avoid any potential confusion, use `make`.
 
 
 ## 2. Create a new `vial` keymap
 
-Copy the existing `keymaps/default` keymap to `keymaps/vial`.
+Copy the existing `keymaps/default` folder to `keymaps/vial` with the content intact. Leave the `keymaps/default` folder in place; do not modify or remove it.
+
+> Important
+> {: .label .label-red }When porting to Vial, the resulting keymap should ALWAYS be named `vial` and nothing else! This avoids any confusion about if the keyboard has already been ported or not.
 
 ## 3. Enable Vial in your rules file
 
-Create a new file under `[keyboard_name]/keymaps/vial/rules.mk` with the following contents:<sup>[(example)](https://github.com/vial-kb/vial-qmk/blob/90f3b0e2e188eccb23ed8a2a690df278a0f1057b/keyboards/vial_example/vial_atmega32u4/keymaps/vial/rules.mk#L2)</sup>
+Create a new file (or edit an existing one) under `[keyboard_name]/keymaps/vial/rules.mk` with the following contents:<sup>[(example)](https://github.com/vial-kb/vial-qmk/blob/90f3b0e2e188eccb23ed8a2a690df278a0f1057b/keyboards/vial_example/vial_atmega32u4/keymaps/vial/rules.mk#L2)</sup>
 
 ```
 VIA_ENABLE = yes
 VIAL_ENABLE = yes
 ```
+> Important
+> {: .label .label-red }
+> Both of these rules are required. Do not remove `VIA_ENABLE`.
+
+Including these in a keyboard level `rules.mk` is not adviced.
```

Comment:
i would just say, do not include at keyboard level, rather than "not advised". since we wouldn't accept any changes made on keyboard level